app/models/publication-flow.js

```diff
@@ -11,6 +12,7 @@ export default class PublicationFlow extends Model {
   @attr('string') remark;
   @attr('datetime') created;
   @attr('datetime') modified;
+  @attr('user') modifiedBy;
```

Comment:
Is geen attr maar een belongsTo
Maar dit moet hier niet staan, die belongsTo is al gedefinieerd in ModelWithModifier.